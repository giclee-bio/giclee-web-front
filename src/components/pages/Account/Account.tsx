import Button from '@/components/atoms/Button'
import AccountAvatar from '@/components/molecules/AccountAvatar'
import AccountInfoForm from '@/components/organisms/AccountInfoForm'

const Account = () => {
  return (
    <article className='rounded px-8 py-12'>
      <h1 className='text-2xl font-bold'>アカウント設定</h1>
      <section className='mt-12'>
        <h2 className='text-xl font-bold'>アカウント情報</h2>
        <div className='mt-8 flex gap-24 px-4'>
          <AccountAvatar />
          <AccountInfoForm />
        </div>
      </section>
      <section className='mt-12'>
        <h2 className='mb-8 text-xl font-bold'>アカウント削除</h2>
        <Button type='danger' width='half'>
          削除する
        </Button>
      </section>
    </article>
  )
}

export default Account
