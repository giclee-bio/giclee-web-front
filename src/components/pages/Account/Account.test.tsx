import { render } from '@testing-library/react'

import Account from './Account'

describe('Account', () => {
  it('snapshot', async () => {
    const { asFragment } = render(<Account />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
