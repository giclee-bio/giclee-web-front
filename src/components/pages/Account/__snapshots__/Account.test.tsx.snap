// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Account snapshot 1`] = `
<DocumentFragment>
  <article
    class="rounded px-8 py-12"
  >
    <h1
      class="text-2xl font-bold"
    >
      アカウント設定
    </h1>
    <section
      class="mt-12"
    >
      <h2
        class="text-xl font-bold"
      >
        アカウント情報
      </h2>
      <div
        class="mt-8 flex gap-24 px-4"
      >
        <div>
          <button>
            <img
              alt="aa"
              class="rounded-full"
              data-nimg="1"
              decoding="async"
              height="80"
              loading="lazy"
              src="/_next/image?url=%2Fnacal.png&w=256&q=75"
              srcset="/_next/image?url=%2Fnacal.png&w=96&q=75 1x, /_next/image?url=%2Fnacal.png&w=256&q=75 2x"
              style="color: transparent;"
              width="80"
            />
            <p
              class="mt-2 text-sm"
            >
              変更する
            </p>
          </button>
        </div>
        <form
          class="w-64"
        >
          <div
            class="mt-4 first:mt-0"
          >
            <div
              class="flex justify-between text-sm"
            >
              <label
                class="font-bold"
                for="radix-:r0:"
              >
                表示名
              </label>
            </div>
            <input
              class="mt-2 bg-stone-50 h-8 w-full rounded px-3 text-black"
              id="radix-:r0:"
              name="displayName"
              required=""
              title=""
              type="text"
              value=""
            />
          </div>
          <div
            class="mt-4 first:mt-0"
          >
            <div
              class="flex justify-between text-sm"
            >
              <label
                class="font-bold"
                for="radix-:r1:"
              >
                自己紹介
              </label>
            </div>
            <textarea
              class="mt-2 bg-stone-50 w-full rounded px-3 text-black"
              cols="30"
              id="radix-:r1:"
              name="username"
              required=""
              rows="5"
              title=""
            />
          </div>
          <button
            class="mt-9 block  h-10 rounded text-center font-bold leading-10 w-32 bg-primary text-black"
          >
            更新する
          </button>
        </form>
      </div>
    </section>
    <section
      class="mt-12"
    >
      <h2
        class="mb-8 text-xl font-bold"
      >
        アカウント削除
      </h2>
      <button
        class="h-10 rounded text-center font-bold leading-10 w-32 bg-ruby-400 text-black"
      >
        削除する
      </button>
    </section>
  </article>
</DocumentFragment>
`;
